### ISSUE TITLE
Travis CI not running

### ISSUE THREAD

**Issue by LecrisUT** [conda-forge/spglib-feedstock#92]
We constantly get the error `We are unable to start your build at this time. You exceeded the number of users allowed for your plan. Please review your plan details and follow the steps to resolution. ` when a build is being triggerred by a pus or PR. Occasionally when settings are changed, the status is changed so that it can be triggerred manually, but that is not a solution.

@conda-forge/core

**Comment by beckermr** [comment]
Which settings are changed?

**Comment by LecrisUT** [comment]
Clarification, when I manually change the repo settings, e.g. `User Management`, or just reload the page

**Comment by beckermr** [comment]
you have permissions to change the repo settings on travis for your feedstocks?

**Comment by LecrisUT** [comment]
The UI allows it, although some settings are note persistent (`User Management` I get an error there). I can Limit concurrent jobs and such though

**Comment by beckermr** [comment]
ok thanks - we will write travis support about the error

**Comment by jakirkham** [comment]
Yes they did and fixed the issue. Just trying to make sure it doesn't reoccur.

**Comment by LecrisUT** [comment]
Could you share if this was an issue on the conda side, travis side, or exactly what it said in that travis only provided a certain number of users?
